src/kernel/include/thd.h

```diff
@@ -75,7 +77,7 @@ struct thread {
 	struct list        event_head; /* all events for *this* end-point */
 	struct list_node   event_list; /* the list of events for another end-point */
 } CACHE_ALIGNED;
-
+#include "fpu.h"
```

Comment:
Does this also have to be included here for some reason? Can it be included up with the rest of the includes at the top of the file?

Reply:
It is connected with the problem of fpu.h file. No, it cannot be with other includes.